src/cheri_insts.sail

```diff
@@ -189,6 +189,13 @@ function clause execute (CSpecialRW(cd, scr, cs1)) =
 }
 
 union clause ast = CAndPerm : (regidx, regidx, regidx)
+/*!
+ * Capability register *cd* is replaced with the contents of capability
+ * register *cs1* with the \cperms{} field set to the bitwise-AND of
+ * its previous value and bits 0 .. 10 of integer register *rs2*
```

Comment:
This isn't actually true any more; CID is bit 11 (and ASR is bit 10, unclear if `0 .. 10` includes 10 too or not, but probably?).

Reply:
We should probably use a macro to avoid bugs like this.

Reply:
Now that the documentation is part of the Sail I think it's fine.

Reply:
Maybe best to say something like `the \cperms{} and \cuperms{} fields bitwise-ANDed with their joint architectural representation (see \cref{..., ...}) in integer register *rs2*`?

Reply:
As I discussed with Alex on Slack, these docs need to make sense standalone, so no `\cref`'s to cheri-architecture places (especially since they're prone to being renamed without realising this other repository needs to be updated).

Reply:
That's fair... but can we not cross-reference saildocs?

Reply:
Currently cheri-architecture has no label for the documentation subsection, only one for the function clause itself and one for the whole instruction, so you'd (a) want to reference one of those two (b) use a suitable macro to abstract cheri-architecture specifics away.

Reply:
Well, I'd meant more like pointing at the sail helpers `getCapPerms` and `setCapPerms` but while that's _named_ in 7.3 of the ISA doc, there's no analogous point in chapter 8, and even in 7.3 it's just a name, not the actual layout.